### PR TITLE
Fix action can't run on centos6

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

**Review by Aiee — APPROVED** [review]

File: .github/actions/upload-to-oss-action/action.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+              folder_list[j]=${{ inputs.asset-path }}/$filename
+              j=`expr $j + 1`
+          done
+        else
+          folder_list[0]=${{ inputs.asset-path }}
+        fi
+        echo "Uploading to oss... "
+        for filepath in ${folder_list[@]};
+        do
+            ossutil64 -e ${{ inputs.endpoint}} \
```
Comment (laura-ding) [inline .github/actions/upload-to-oss-action/action.yml:44]: Unified the indentation

File: .github/actions/upload-to-oss-action/action.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    required: true
+    default: ''
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        j=0
+        if [ -d ${{ inputs.asset-path }} ]; then
+          for filename in `ls ${{ inputs.asset-path }}`; do
+              folder_list[j]=${{ inputs.asset-path }}/$filename
```
Comment (laura-ding) [inline .github/actions/upload-to-oss-action/action.yml:35]: Unified the indentation

File: .github/actions/upload-assets-action/action.yml [hunk 3]
```diff
@@ -15,13 +15,25 @@ runs:
     - run: |
         GH_RELEASE="https://api.github.com/repos/${{ github.repository }}/releases/tags/${{ inputs.tag }}"
         upload_url=$(curl -s --request GET --url $GH_RELEASE | grep -oP '(?<="upload_url": ")[^"]*' | cut -d'{' -f1)
-        content_type=$(file -b --mime-type ${{ inputs.file-path }})
-        filename=$(basename "${{ inputs.file-path }}")
+        j=0
+        if [ -d ${{ inputs.asset-path }} ]; then
+          for filename in `ls ${{ inputs.asset-path }}`; do
+              folder_list[j]=${{ inputs.asset-path }}/$filename
```
Comment (laura-ding) [inline .github/actions/upload-assets-action/action.yml:21]: Unified the indentation

File: .github/actions/upload-to-oss-action/action.yml (hunk 1 above)
Comment (jievince) [reply .github/actions/upload-to-oss-action/action.yml:45]: unified.

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -45,17 +45,16 @@ jobs:
           echo "::set-output name=subdir::$subdir"
```
Comment (yixinglu) [inline .github/workflows/release.yml:45]: These output vars seem useless, could you check whether we should delete them?

File: .github/workflows/package.yml [hunk 5]
```diff
@@ -38,16 +38,15 @@ jobs:
           echo "::set-output name=filepath::$filename"
           echo "::set-output name=shafilepath::$filename.$SHA_EXT"
           echo "::set-output name=subdir::$subdir"
```
Comment (yixinglu) [inline .github/workflows/package.yml:40]: See the below comment. thx.

**Review by yixinglu — COMMENTED** [review]
Nice PR

File: .github/workflows/release.yml (hunk 4 above)
Comment (jievince) [reply .github/workflows/release.yml:43]: You're right, they are redundant.

File: .github/workflows/package.yml (hunk 5 above)
Comment (jievince) [reply .github/workflows/package.yml:38]: fixed

File: .github/workflows/release.yml (hunk 4 above)
Comment (jievince) [reply .github/workflows/release.yml:43]: `subdir` is used in `target-path`. `filepath` and `shafilepath` are not used now so delete them

**Review by yixinglu — APPROVED** [review]

**Review by laura-ding — APPROVED** [review]

**Pull request merged**